src/compiler/checker.ts

```diff
@@ -9023,6 +9023,9 @@ namespace ts {
             if (symbol === requireSymbol) {
                 return anyType;
             }
+            if (!symbol.valueDeclaration && symbol.escapedName === "meta") {

```

Comment:
I don’t think this is a sufficient condition. This is a really weird case, because there actually is no symbol for `import.meta`, but because completions works with symbols, it creates a dummy symbol for it on line 1593 (and presumably the same crash happens with `new.target`). The least we can do is keep this kludge private to `completions.ts` and not ask the checker for the type of a completely made-up symbol with literally no context. The checker knows how to get the proper type for `import.meta` and `new.target` from a _node_, so I would suggest making `completions.ts` somehow aware that it needs to use a different method to get the type of these, and able to trace these dummy symbols back to the nodes they came from.

Reply:
I thought it was a strange looking symbol as well, but didn't investigate where it was from. Yes, this would also happen for `new.target`.

That makes more sense and it's better not to use this dummy symbol. I've updated the PR with a change.